eng/build.ps1

```diff
@@ -382,7 +382,7 @@ function TestUsingRunTests() {
   $args += " --configuration $configuration"
 
   if ($testCoreClr) {
-    $args += " --tfm net6.0"
+    $args += " --tfm net6.0 --tfm net7.0"
```

Comment:
From discussion with Jared, RunTests uses this list of TFMs to scan the folders it is given as input, and it already knows how to run the discovered tests in a given TFM with the corresponding runtime.
Since no unittest currently builds *both* for net6.0 and net7.0 TFMs, we're not introducing duplication.

Reply:
Please confirm by observation that both sets of tests are run in CI and locally (build -testCoreClr)

Reply:
One way to do that is to make some test fail and observe that during test run.

Reply:
Yes, I've verified that an intentional failure in a compiler test and in a workspace test get reported when running locally. I can do the same in CI.
For CI verification, I'm planning to compare the total number of tests executed in this PR against previous PRs.

Reply:
The expected failures showed up in CI and the total count matched previous PRs.
![image](https://user-images.githubusercontent.com/12466233/194914512-ae07e996-51d5-486e-bd28-4694390a40da.png)